src/index.js

```diff
@@ -39,13 +39,19 @@ export default class Rollup extends Plugin {
     this.lastTree = FSTree.fromEntries([]);
     this.linkedModules = false;
     this.cache = options.cache === undefined ? true : options.cache;
+
+    if (options.nodeModulesPath && !path.isAbsolute(options.nodeModulesPath)) {
```

Comment:
should this be `'nodeModulesPath' in options` ? That way an accidental undefined doesn't fallback tow CWD.